manifests/init.pp

```diff
@@ -25,6 +26,15 @@
   $globaloptions       = '',
   $omapi_port          = undef,
   $extra_config        = '',
+  $peer_address        = undef,
+  $role                = undef,
+  $port                = undef,
+  $max_response_delay  = undef,
+  $max_unacked_updates = undef,
+  $mclt                = undef,
+  $load_split          = undef,
+  $load_balance        = undef,
+  $omapi_key           = undef,
```

Comment:
This parameter appears to be unused. Maybe it's better to remove it here and fix it through https://github.com/voxpupuli/puppet-dhcp/issues/90